src/NHibernate.Test/NHSpecificTest/NH3046/Model.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace NHibernate.Test.NHSpecificTest.NH3010
```

Comment:
Sorry, made a mistake with the files (I'm new on Git).

Will upload the changes soon.